src/components/RadioButtons/RadioButtons.jsx

```diff
@@ -49,7 +51,21 @@ class RadioButtons extends React.Component {
   }
 
   handleChange(domEvent) {
-    this.props.onValueChange(makeField(domEvent.target.value, true));
+    const clickLabel = domEvent.target.value;
+
+    if (this.props.enableAnalytics) {
+      dispatchAnalyticsEvent({
+        componentName: 'RadioButtons',
+        action: 'change',
+
+        details: {
+          clickLabel: clickLabel,
+          label: this.props.label,
```

Comment:
@bsmartin-ep Check out my comment [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14493#issuecomment-815295838) regarding the option label values